projects/npm-tools/packages/npm-scripts/src/jsp/checkExposesConfiguration.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/* eslint-disable no-console */
```

Comment:
Lint suppressions should be made as locally as possible (see [guidelines](https://github.com/liferay/liferay-frontend-projects/blob/master/guidelines/dxp/linting.md#lint-suppressions)), because it's too easy for an overly broad suppression to end up allowing in actually-bad stuff unintentionally.

In this case, you don't even have any `console` usages in this file, so it's not even a question of scoping this more narrowly; just remove it.

Reply:
:-( I forgot to remove old code because this was a standalone script before the refactor. I will remove it. This applies to some other revisions below as well.

Sorry.